doc/tasks.rst

```diff
@@ -168,6 +168,25 @@ check :ref:`tools.LongRunning<tools.LongRunning>` as an example.
 
 
 
+action arguments
+-----------------
+
+You always can pass some arguments into your action in case it's a `python-action`.
```

Comment:
dont need repeat this explanation. It is already said on the section above `python-actions`. You only need to add an example that uses kwargs.